### DISCUSSION TITLE
What am I doing wrong?

### DISCUSSION THREAD

**Discussion by koenbunders** [mdelobelle/metadatamenu#216] [Q&A]
I have a fileClass “Project” and in it a field “Life Area”.
Using Metadata Menu plugin I try to populate the options for that field from all pages that I have with the #Area tag, see screenshot. The query doesn’t return a thing. Why?
![why not working](https://user-images.githubusercontent.com/12957034/209552565-23f2ead0-38f1-4d71-870f-73c3b412970b.jpg)

**Comment by koenbunders** [comment]
@mdelobelle any suggestion?

**Reply by mdelobelle** [reply]
Hi! the field is expecting a list of string. your query is returning a list of dataview pages.
could you try with dv.pages('#Area').map(p=>p.file.name)?

If you wan't to have the links to those files, consider using the Field type ("accept a link"), with this type of field, your query should be valid
